www/front_src/src/Resources/Details/tabs/Timeline/ExportToCsv.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslation } from 'react-i18next';
+import { useAtomValue } from 'jotai';
+
+import { Stack } from '@mui/material';
+import SaveAsImageIcon from '@mui/icons-material/SaveAlt';
+
+import { labelExportToCSV } from '../../../translatedLabels';
+import { detailsAtom } from '../../detailsAtoms';
+import ResourceActionButton from '../../../Actions/Resource/ResourceActionButton';
+
+const ExportToCsv = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  const details = useAtomValue(detailsAtom);
+  const downloadTimelineFile = `${details?.links.endpoints.timeline}/download`;
+
+  const exportToCsv = (): void => {
+    window.open(downloadTimelineFile, 'noopener', 'noreferrer');
+  };
+
+  return (
+    <Stack
+      alignItems="center"
+      direction="row"
+      justifyContent="flex-end"
+      spacing={0.5}
+    >
+      <ResourceActionButton
```

Comment:
The ResourceActionButton is mainly use because of the responsiveness it handles. I think a regular button is enough here